WordPressAuthenticator/Analytics/AuthenticatorAnalyticsTracker.swift

```diff
@@ -142,6 +142,10 @@ public class AuthenticatorAnalyticsTracker {
         ///
         case continueWithWordPressCom = "continue_with_wordpress_com"
 
+        /// Tracked when the user clicks “What is WordPress.com?" button on the WordPress.com flow screen
+        ///
+        case whatIsWpCom = "what_is_wordpress_com"
```

Comment:
super nit: it'd be nice having a consistent naming convention
```suggestion
        case whatIsWPCom = "what_is_wordpress_com"
```

Reply:
It might be worth confirming with WPiOS about adding a new event to the analytics tracker, I'm not super sure if they have a workflow for analytics like whether they keep track of all the events. @pmusolino do you know who we can reach out for taking a look at this, after the recent team changes in WPiOS? (I saw that a bunch of the authors aren't with WPiOS anymore)

Reply:
> super nit: it'd be nice having a consistent naming convention

Done in 48fbd1713d510b8140fb2ebd3dd0b55e20a7df2f

Reply:
> It might be worth confirming with WPiOS about adding a new event to the analytics tracker, I'm not super sure if they have a workflow for analytics like whether they keep track of all the events. @pmusolino do you know who we can reach out for taking a look at this, after the recent team changes in WPiOS? (I saw that a bunch of the authors aren't with WPiOS anymore)

I haven't had success with building https://github.com/wordpress-mobile/WordPress-iOS app. I will try to build again and validate this.

Reply:
I have raised a PR in WP iOS https://github.com/wordpress-mobile/WordPress-iOS/pull/18996 to test this. 

I have also validated that the changes related to the analytics tracker are non-breaking.

Reply:
Yea I don't think the change affects WPiOS, but just wanted to see if they have a centralized doc to keep track of the events